Extensions/SendTo/ClientPickerViewController.swift

```diff
@@ -47,7 +47,13 @@ class ClientPickerViewController: UITableViewController {
         title = NSLocalizedString("Send Tab", tableName: "SendTo", comment: "Title of the dialog that allows you to send a tab to a different device")
         refreshControl = UIRefreshControl()
         refreshControl?.addTarget(self, action: #selector(ClientPickerViewController.refresh), forControlEvents: UIControlEvents.ValueChanged)
-        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.Cancel, target: self, action: #selector(ClientPickerViewController.cancel))
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: NSLocalizedString("Cancel", tableName: "SendTo", comment: "Button title for cancelling SendTo screen"),
```

Comment:
Can this be updated to use `value` too? Re: IRC discussions, we can try using `CamelCase.WithDotsBetween.Label` now. Also, it sounds like we don't want to add `tableName`s to new strings.

Going to try to get this all sorted out soon in [bug 1261431](https://bugzilla.mozilla.org/show_bug.cgi?id=1261431)!